libraries/botframework-connector/src/teams/operations/teams.ts

```diff
@@ -298,6 +300,210 @@ export class Teams {
             callback
         ) as Promise<Models.TeamsMeetingNotificationResponse>;
     }
+
+    //Batch Operations
+    /**
+     * Send message to a list of users.
+     * 
+    /**
+     *
+     * @param activity The activity to send.
+     * @param tenantId The tenant Id.
+     * @param members The recipient members list.
+     * @param options The optional parameters.
+     * @param callback The callback.
+     * @returns Promise with TeamsBatchOperationResponse.
+     */
```

Comment:
I think this the middle /** shouldnt go
```suggestion
    /**
     * Send message to a list of users.
     *
     * @param activity The activity to send.
     * @param tenantId The tenant Id.
     * @param members The recipient members list.
     * @param options The optional parameters.
     * @param callback The callback.
     * @returns Promise with TeamsBatchOperationResponse.
     */
```

same in other cases